### PR TITLE
Fix header guards for include/acacia/atspi

### PR DESCRIPTION
**Pull request by alice**

### CODE REVIEW

File: lib/atspi/atspi_node.cc [hunk 1]
```diff
@@ -394,7 +394,8 @@ AtspiDocumentInterface AtspiNode::queryDocument() const {
 AtspiHyperlinkInterface AtspiNode::queryHyperlink() const {
   // Unlike the other interfaces, `atspi_accessible_get_hyperlink` gives us a
   // new hyperlink object that serves as the interface. Therefore we cannot
-  // free it until we're done with it. See also `atspi_hyperlink_interface.h`.
+  // free it until we're done with it, so we pass ownership to the
+  // AtspiHyperlinkInterface object. See also `atspi_hyperlink_interface.h`.
```
Comment (spectranaut) [inline lib/atspi/atspi_node.cc:398]: These seems unrelated, and I see this line is removed in: https://github.com/Igalia/acacia/pull/210/files#diff-622c94f9ea8c0f81ccfd47ede4504ed3150af66b1377b72f129a1962361e9b0c

**Review by spectranaut — APPROVED** [review]
Thanks for this slightly tedious fix! There is one unrelated change, but if you remove it you will have a conflict with the next PR in this chain: https://github.com/Igalia/acacia/pull/210

so if you want to leave it in that's fine.

**Comment by spectranaut** [comment]
want to switch the base branch to main and land?

File: lib/atspi/atspi_node.cc (hunk 1 above)
Comment (alice) [reply lib/atspi/atspi_node.cc:398]: Thanks for catching, will fix and land!
